### DISCUSSION TITLE
Reactivating an ignored PR in Azure DevOps doesn't work when using the provided directions for renaming

### DISCUSSION THREAD

**Discussion by sheldonhull** [renovatebot/renovate#21700] [Suggest an Idea]
### How are you running Renovate?

WhiteSource Renovate hosted app on github.com

### Please select which platform you are using if self-hosting.

Azure DevOps (dev.azure.com)

### Describe the bug

I'm using the renovate CLI to scan dependencies and open PR's while using the Azure DevOps service. 
This pipeline works great and creates all my PR's with changes. 

## Abandoned (Closed) PR

I have a "stalebot" style tool cleaning up PR's that have been sitting idle. 

I went to reactivate one but found that this won't work in Azure DevOps as the branch is removed as part of cleanup. Because of this I need to follow the directions it said below: 

```
Renovate Ignore Notification
As this PR has been closed unmerged, Renovate will ignore this upgrade 
and you will not receive PRs for any future REPO  digest updates. 
Digest updates will resume if you update the specified tag at any time.

If this PR was closed by mistake or you changed your mind, 
you can simply rename this PR and you will soon get a fresh replacement PR opened.
```

This is problematic in Azure DevOps because once a PR is merged or abandoned (their term for closed without merge) there is no ability to rename the PR, and you can't reopen if the branch no longer exists.

## Issue

This makes the current reopen option unviable.

I'd request a method so I can reopen an abandoned PR and for Azure DevOps ensure the closing message properly describes how to "UN-Ignore" the prior abandoned PR. 

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by viceice** [comment]
Similar to
- #6552

**Comment by gioce90** [comment]
There are two workarounds here: https://github.com/renovatebot/renovate/discussions/10456
